paypal/express/views.py

```diff
@@ -388,6 +388,46 @@ def get_shipping_method(self, basket, shipping_address=None, **kwargs):
 
 class ShippingOptionsView(View):
 
+    def get(self, request, *args, **kwargs):
+        """
+        We use the shipping address given to use by PayPal to
+        determine the available shipping method
+        """
+        # Basket ID is passed within the URL path.  We need to do this as some
+        # shipping options depend on the user and basket contents.  PayPal do
+        # pass back details of the basket contents but it would be royal pain to
+        # reconstitute the basket based on those - easier to just to piggy-back
+        # the basket ID in the callback URL.
+        basket = get_object_or_404(Basket, id=kwargs['basket_id'])
+        user = basket.owner
+        if not user:
+            user = AnonymousUser()
+
+        logger.info(kwargs['country_code'])
```

Comment:
Is this for debug purposes? We'd remove this logger call anyway.

Reply:
I didn't realise anyone was watching the changes as this PR was made in 2017, these changes were made for a specific project and I needed to monitor in production. Will clean up and push changes.

Reply:
removed.